py/specs/select/group_by_no_doc_values.toml

```diff
@@ -0,0 +1,37 @@
+[setup]
+statement_files = ["sql/uservisits_no_doc_values.sql"]
+statements = [
+    "copy uservisits_no_doc_values from 's3://crate.amplab/data/1node/uservisits/part-00001*' with (compression = 'gzip')",
+    "copy uservisits_no_doc_values from 's3://crate.amplab/data/1node/uservisits/part-00002*' with (compression = 'gzip')",
+    "copy uservisits_no_doc_values from 's3://crate.amplab/data/1node/uservisits/part-00003*' with (compression = 'gzip')",
+    "copy uservisits_no_doc_values from 's3://crate.amplab/data/1node/uservisits/part-00004*' with (compression = 'gzip')",
+    "copy uservisits_no_doc_values from 's3://crate.amplab/data/1node/uservisits/part-00005*' with (compression = 'gzip')",
+    "refresh table uservisits_no_doc_values"
+]
+
+[[queries]]
+statement = '''select "cCode", count(*) from uservisits_no_doc_values group by "cCode"'''
+iterations = 100
+
+[[queries]]
+statement = '''select min("adRevenue") from uservisits_no_doc_values group by "cCode"'''
```

Comment:
Does it make sense to test so many different cases? 
I'd go for one case which accesses/uses the values for each row (avg for example), and that is the opposite (arbitrary?)

I think that would be enough to give us a good feeling for the performance of with doc-values vs. without.
I'm also not sure if it's necessary to have this both as global aggregate and as group by test - since the "cCode" column has docvalues